cfgov/core/services.py

```diff
@@ -0,0 +1,189 @@
+import os
+
+import six
+import requests
+import icalendar
+from django.views.generic.base import View, ContextMixin
+from django.http import HttpResponse, Http404
+from django.core.exceptions import ImproperlyConfigured
+from pytz import timezone
+from dateutil.parser import parse
+
+# PDFreactor's python wrapper doesn't support python 3, so neither can we for
```

Comment:
Worth adding a `TODO:` comment? E.g. `# TODO: Update to python 3 when PDFreactor's python wrapper supports it`